imxvpuapi/imxvpuapi.h

```diff
@@ -101,25 +101,25 @@ typedef struct _ImxVpuDMABufferAllocator ImxVpuDMABufferAllocator;
  * The vfuncs are:
  *
  * allocate(): Allocates a DMA buffer. "size" is the size of the buffer in bytes. "alignment" is the address
- *             alignment in bytes. An alignment of 1 or 0 means that no alignment is required.
- *             "flags" is a bitwise OR combination of flags (or 0 if no flags are used, in which case
- *             cached pages are used by default). See ImxVpuAllocationFlags for a list of valid flags.
- *             If allocation fails, NULL is returned.
+ *			 alignment in bytes. An alignment of 1 or 0 means that no alignment is required.
+ *			 "flags" is a bitwise OR combination of flags (or 0 if no flags are used, in which case
+ *			 cached pages are used by default). See ImxVpuAllocationFlags for a list of valid flags.
+ *			 If allocation fails, NULL is returned.
  *
```

Comment:
Here, also keep the spaces please. They are used for alignment here. Same goes for the spaces/tabs _within_ the comment (tabs before the comment lines are fine, as they are then used for indentation).